releasenotes/notes/0.19/fix-decompose-empty-gate-71455847dcaaea26.yaml

```diff
@@ -0,0 +1,4 @@
+---
+fixes:
+  - |
+    Fixed a bug where circuit gates whose definitions were empty circuits were not decomposed.
```

Comment:
I've put this in, but I'm not sure I agree that we need to do this.  If you'd seen the bug then it'd make sense to you, otherwise it doesn't really tell you all that much, and makes it more daunting to read the release notes, because they go on for screens and screens.  I'm more of a fan of just linking to PRs/issues for bug fixes.